util/scripts/sync_tas/sync_pi_status_from_tas.py

```diff
@@ -0,0 +1,126 @@
+import os
+import sys
+import logging
+import django
+
+# loading up django so we can use models for queries/updates
+sys.path.append('/project') 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "chameleon.settings")
+django.setup()
```

Comment:
What does this do?

Reply:
this loads django so we can run this as a script independent of the django instance managed by uwsgi (so we don't have to create a url to reach the logic or worry about timeouts) and still use the django models api to read/write to the db